src/main.rs

```diff
@@ -160,3 +152,30 @@ fn pass_to_sub(command: String) -> Result<(), MainError> {
         None => MainError::Pn(PnError::UnexpectedTermination { command }),
     })
 }
+
+fn get_prepended_path_env(prepend_path: &str) -> OsString {
+    if let Some(path) = env::var_os("PATH") {
+        let mut new_paths: Vec<PathBuf> = vec![PathBuf::from(prepend_path)];
+        let mut current_paths = env::split_paths(&path).collect::<Vec<_>>();
+        new_paths.append(&mut current_paths);
+
+        let new_path = env::join_paths(new_paths.iter()).expect("Failed to prepend path");
+        new_path
+    } else {
+        OsString::from(prepend_path)
+    }
+}
+
+#[test]
+fn test_get_prepended_path_env() {
+    #[allow(unused_imports)]
+    use std::path::PathBuf;
+
+    let bin_dir = "node_modules/.bin";
+    let prepended_path_env = get_prepended_path_env(bin_dir);
+
+    // Check that the node_modules directory was added to the PATH variable
```

Comment:
Comment that describes what the code does is a bad comment.

A good comment explains the _why_, not the _what_.

Anyway, this is a bad comment.

```suggestion
```